test/videojs-flashls-source-handler.test.js

```diff
@@ -35,3 +57,64 @@ QUnit.module('videojs-flashls-source-handler', {
   }
 });
 
+QUnit.test('Flashls Audiotracks tests',
+  function(assert) {
+    const swfAudioTracks = [
+      {
+        id: 0,
+        enabled: true,
+        kind: '',
+        label: 'English',
+        language: ''
+      },
+      {
+        id: 1,
+        enabled: false,
+        kind: '',
+        label: 'Spanish',
+        language: ''
+      },
+      {
+        id: 2,
+        enabled: false,
+        kind: '',
+        label: 'French',
+        language: ''
+      }
+    ];
+
+    let vjsAudioTracks = [];
+    let swfAudioTrack = 0;
+
+    const tech = makeMochTech({
+      audioTracks: () => swfAudioTracks,
+      audioTrack: () => swfAudioTrack
+    }, {
+      audioTrack: (val) => swfAudioTrack = val
+    });
+
+    tech.audioTracks = () => {
+      return vjsAudioTracks;
+    };
+
+    tech.audioTracks().addTrack = (t) => {
+      vjsAudioTracks.push(t);
+    };
+
+    tech.audioTracks().tracks_ = vjsAudioTracks;
```

Comment:
after you make the change to `updateAudioTrack` i proposed, you won't need to have this line.